Challenge 44&45: Vault Metadata challenge (#1147)
OWASP/wrongsecrets: src/main/resources/explanations/challenge45.adoc

=== Vault subkey challenge

> Sometimes, all you want to do, is having that concise entry in your secrets management system. So what about storing your usnerame and password in the same entry?
> We tried doing that, but got into a new problem! Because with Hashicorp Vault you can setup policies to allow access to a subkey (Which is the key to the value of your secret). Can you find the very random username we setup in this challenge?

Reviewer: ```suggestion
Sometimes, all you want to do is have that concise entry in your secrets management system. So, what about storing your username and password in the same entry?
We tried doing that but got into a new problem! With Hashicorp Vault, you can set up policies to allow access to a subkey (Which is the key to the value of your secret). Can you find the very random username we set up for this challenge?
```
